aten/src/ATen/native/Resize.h

```diff
@@ -51,15 +52,7 @@ inline TensorImpl* resize_impl_cpu_(
   if (stride) {
     self->set_sizes_and_strides(size, *stride);
     // NB: storage size can be different from numel.
-    for (size_t dim = 0; dim < size.size(); ++dim) {
-      // FIXME: Don't rely on storage_size being negative because this
```

Comment:
I see that this comment was dropped. Do you know what it was referring to?

Reply:
I wrote this comment two years ago but unfortunately I don't remember why :/

Looking at [the original PR](https://github.com/pytorch/pytorch/pull/13185/files#diff-67630b8526b429e0fc2c7851f15eb290808c90d898e8ab98fb28821d513c3a46R38), previously, storage_size could be negative if size[dim] is equal to 0. I think that is what the comment is referring to.

However, in the current iteration of the code, storage_size can never be negative, so I think the comment doesn't apply anymore.

Reply:
Right the dim=0 case is special cased so that should be fine!